README.org

```diff
@@ -0,0 +1,42 @@
+#+title: Alejandro_Vyacheslav_Microverse_Trial_cool_project
+#+auto-tangle: t
+
+* DEPLOYMENT IN GITHUB PAGES
+See the ready Deployed project on [ Github Pages](https://hacking-nassa-with-html.github.io/Alejandro_Vyacheslav_Microverse_Trial_cool_project/).
+* Code Rank Exercises
+** Minimum Distances (Problem 01)
+[[https://www.hackerrank.com/challenges/minimum-distances/problem][Link to problem in HackerRank]]
+Solution:
+#+begin_src js :tangle generatedByREADME/solution01.js
+// This file was generated by README.org, read the solution from the source to get the link to the problem
+function minimumDistances(a) {
+    // Write your code here
+    const minDistances = [];
+
+    for (let i = 0; i < a.length; i++) {
+        for (let j = i+1; j < a.length; j++) {
+            if (a[i] === a[j]) {
+                minDistances.push(Math.abs(i-j));
+            }
+        }
+    };
+
+    return minDistances.length > 0 ? Math.min(...minDistances): -1;
+}
+#+end_src
+** The Hurdle Race (Problem 02)
+[[https://www.hackerrank.com/challenges/the-hurdle-race/problem][Link to problem in HackerRank]]
+Solution:
+#+begin_src js :tangle generatedByREADME/solution02.js
+// This file was generated by README.org, read the solution from the source to get the link to the problem
+function hurdleRace(k, height) {
+    // Write your code here
+  let maximum = height.reduce((max, element) => {
+    if(element > max) return element;
+    return max;
+  });
+
+  let numPotions = (maximum > k) ? (maximum - k) : 0;
+  return numPotions;
+}
+#+end_src
```

Comment:
Nice detail but you must follow the README template from Microverse.
Please, rename this file to CodeChallenges.org and include the README template or Replicate of the README template from Microverse as an .org file instead of .md, and add this as an org bullet section with a proper title.

Reply:
Well done on adding inline comments 💪, but we always need to let the student know why we asked for a change otherwise it will sound like a personal opinion , providing arguments and links to resources will help them learn and make our review more professional 😊

Reply:
Nice detail but you must follow the README template from Microverse.
The README file for your project should follow the Microverse README Template 
- please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

This is important because it is one of the 5 basic rules to follow in Microverse projects to make your project looks more professional for future employers

So please,  rename this file to CodeChallenges.org and include the README template or Create a Replicate of the README template from Microverse as an .org file instead of .md, and add this as an org bullet section with a proper title.

Reply:
Nice detail but you must follow the README template from Microverse.
The README file for your project should follow the Microverse README Template

please use this template but remember to customize it to your project.

This is important because it is one of the 5 basic rules to follow in Microverse projects to make your project looks more professional for future employers

So please, rename this file to CodeChallenges.org and include the README template or Create a Replicate of the README template from Microverse as an .org file instead of .md, and add this as an org bullet section with a proper title.

For more information: https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md